### ISSUE TITLE
Create a slot for a "drag handle"?

### ISSUE THREAD

**Issue by youraerials** [bensladden/vue-responsive-dash#139]
Hi, we're enjoying this library, thanks for creating it!  We need to make only the top part of a dash item draggable because we have other UI going on inside the item.  What I did as a hack was just to select the drag listener for each dash item in CSS and resize it like so:
```
.item > div:first-of-type {
    bottom: auto !important;
    height: 24px !important;
}
```

This works, but I would rather be able to define specific styles with a slot for the drag handle like we can currently do with the resize slots.  Seem like a heavy lift?

Thanks again!

**Comment by bensladden** [comment]
Hi,

Thanks for feedback, yes as i will be swapping over to interactjs soon. When done i can create a prop (possibly slot not sure yet) where you can declare the item you wish to be the draggable area.

**Comment by youraerials** [comment]
Sounds great, will keep our eyes out for the change!

**Comment by bensladden** [comment]
Notes to self: [https://interactjs.io/docs/action-options/#allowfrom-(handle)](https://interactjs.io/docs/action-options/#allowfrom-(handle))

**Comment by bensladden** [comment]
This is now available in v0.4.2. let me know if there are any issues
